spec/dummy/app/assets/config/manifest.js

```diff
@@ -0,0 +1,3 @@
+//= link_tree ../images
```

Comment:
Expected space or tab after '//' in comment      spaced-comment<br>Definition for rule 'vue/max-len' was not found  vue/max-len